src/views/layouts/main.hbs

```diff
@@ -16,13 +16,10 @@
   <meta property="og:type" content="website">
   <meta property="fb:app_id" content="2233419873654139">
 
-  <link rel="dns-prefetch" href="https://use.fontawesome.com">
-  <link rel="dns-prefetch" href="https://code.jquery.com">
-  <link rel="dns-prefetch" href="https://cdnjs.cloudflare.com">
-  <link rel="dns-prefetch" href="https://stackpath.bootstrapcdn.com">
   <link rel="dns-prefetch" href="https://s3.amazonaws.com">
+  <link rel="dns-prefetch" href="https://unpkg.com">
```

Comment:
I just wondered what these are and went to mdn 🙂 
Can/should we also use `preconnect` as [described here](https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch)

Btw, what do we need S3 for?

Reply:
@pheeria no idea about s3, I kept it there just in case. 
But I agree about preconnect. Will add it later.